central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -41,6 +44,26 @@ type sentinel struct {
 	azlogsClient azureLogsClient
 }
 
+func (s sentinel) SendAuditMessage(ctx context.Context, msg *v1.Audit_Message) error {
+	if !features.MicrosoftSentinelNotifier.Enabled() {
+		return nil
+	}
+
+	if !s.notifier.GetMicrosoftSentinel().GetAuditLogDcrConfig().GetEnabled() {
```

Comment:
nit:  should we just call `AuditLoggingEnabled()` here?

Reply:
Changed it